lib/scrabble.rb

```diff
@@ -15,4 +18,13 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score_with_multipliers(word, multi, doubler = 1)
+    score = 0
+    letter = word.upcase.split("")
+    letter.each_with_index {|v,i|score += (point_values[v] * multi[i]) * doubler}
```

Comment:
This is better than using #each but push yourself to use more advanced enums (like reduce). This line is also a bit long & could benefit from breaking some logic out into different methods.